index.html

```diff
@@ -11,15 +11,21 @@
     <header id="headline">
       <nav class="nav-container">
         <div class="logo-text"><a href="#">Spyda</a></div>
-        <div class="right-logo">
-          <img
-            src="images/menu-icon.png"
-            alt="Menu Icon for toggle on small screens"
-          />
+      <div class = "mobile-menu">
+        <ul class = "nav-menu">
+          <li class = "nav-item"><a href = "#headline">Portfolio</a></li>
+          <li class = "nav-item"><a href = "#section-5">About</a></li>
+          <li class = "nav-item"><a href = "#section-7">Contact</a></li>
+        </ul>
+        <div class="x-icon x-style">
+          <span class = "bar-x"></span>
+          <span class = "bar-x"></span>
+          <span class = "bar-x"></span>
+        
```

Comment:
Kindly follow the margin/paddings as per the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/179738056-03384a82-dc61-4bb1-bed5-6a43add0e460.png) | ![image](https://user-images.githubusercontent.com/33268927/179737955-35514aaa-0fed-4192-90ac-7f72c6307794.png) |